setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='src',
+    packages=find_packages(),
+    version='0.1.0',
+    description='Spam classification machine learning models for Zenodo records and communities.',
+    author='zenodo',
+    license='MIT',
```

Comment:
```suggestion
    name='zenodo-classifier',
    packages=find_packages(where='src'),
    package_dir={"": "src"},
    version='0.1.0',
    description='Spam classification machine learning models for Zenodo records and communities.',
    author='CERN',
    license='MIT',
```

- `name` is just for convention/clarity
- `author` is CERN
- the packages changes are so that you end up with the following package names:

```python
find_packages(where='src')
['features', 'data', 'models', 'visualization']
```

That means that you can import things in the notebooks and code like:

```python
from features.text import clean_words
...

# or
from visualization.graph import make_graph
...
```